bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+            </p>
+        </section>
+        <section id="BlockWiseTransferParameters">
+            <h3>BlockWiseTransferParameters</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:block2SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:block1SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+        <section id="QuickBlockWiseTransferParameters">
+            <h3>QuickBlockWiseTransferParameters</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:qblock2SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:qblock1SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+    </section>
+    <section id="mappings">
+        <h2>Mappings</h2>
+        <p>
+            This section describes the strategies and default values to use
+            protocol specific concepts within the
+            <a href="https://w3c.github.io/wot-architecture/#dfn-interaction-model">WoT Interaction model</a>.
+        </p>
+        <section id="default-mappings">
+            <h3>Default mappings</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Operation</th>
+                        <th>Default Binding</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>td:readProperty</code></td>
```

Comment:
Should this be `<td><code>readproperty</code></td>`?

Reply:
Yes, the operation type values are all lowercase (see https://w3c.github.io/wot-thing-description/#table-operation-types).

Should the operation types be prefixed by the way? In the HTTP binding the `td:` prefix is missing, for example: https://w3c.github.io/wot-binding-templates/bindings/protocols/http/index.html#http-default-vocabulary-terms

Reply:
No need for the prefix, why should they be prefixed? If you mean the ontology, the TD ontology is used without a prefix so all the keywords can (I think MUST) be used without a prefix.

Reply:
https://www.w3.org/2022/wot/td/v1.1 says:

```
        "readproperty": "td:readProperty",
```

So it seems that, in the ontology, it's *not* all lowercase, and you need to use the prefix if you want that spelling...

I'll change it to `readproperty`.